.gitignore

```diff
@@ -16,9 +16,12 @@
 .DS_Store
 .env*
 .env
-
+*.vs
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
 psiturkit/*_turk
+.vs/VSWorkspaceState.json
+.vs/slnx.sqlite
```

Comment:
in my experience with .gitignore, if the files have already been committed to the repo, they won't be removed completely. you may need to search for how to remove those files (should be some terminal commands) from the repo, as i'm still seeing the 'there are conflicts' msg on this PR

Reply:
will do